example/serverless.yml

```diff
@@ -50,7 +51,7 @@ functions:
       - schedule:
           rate: rate(1 minute)
           enabled: false
-      - cloudwatchLog: "/aws/lambda/sls-wshop-dev-hello"
+      # - cloudwatchLog: '/aws/lambda/sls-wshop-dev-hello'
```

Comment:
wasn't sure about this. I was testings using this application and that cloudwatch log does not exist, as one would expect 

I've commented out